src/core/file_sys/archive_backend.h

```diff
@@ -12,6 +12,7 @@
 #include "common/common_types.h"
 #include "common/swap.h"
 #include "core/hle/result.h"
+#include "delay_generator.h"
```

Comment:
Please use full path
```suggestion
#include "core/file_sys/delay_generator.h"
```